MerkzettelMathe3.tex

```diff
@@ -398,30 +414,23 @@
 					\end{pmatrix}$
 				\end{multicols}
 			\end{tiny}
-		\subsection{Matrixnormen}
-			\begin{tiny}
-				\begin{align*}
-					\left|\bspVec\right| &= \sqrt{x_1^2 + ... + x_n^2}\\
-					&... 
-				\end{align*}
-			\end{tiny}
 	\section{Differentialgleichung}
 		\subsection{DGL 1. Ordnung}
 			\subsection{homogen}
 				\begin{compactenum}
 					\item Trennung der Variablen
 					\item y' durch $\frac{dy}{dx}$ ersetzen
 					\item dx auf x Seite umstellen
-					\item Seiten jeweils integrieren ($+k_1, +k_2\rightarrow k_3$)
-					\item Nach y umstellen (jeweils k++)
+					\item Seiten integrieren ($c_1-c_2\rightarrow c$)
+					\item Nach y umstellen
 				\end{compactenum}	
 			\subsubsection{Variation der Konstanten (inhomogen)}
 				Wenden wir an wenn wir die Variablen nicht geteilt bekommen:\\
 				$y'+b(x)\cdot y=0$ für $y_1$\\
 				$y'+b(x)\cdot y=a(x)$ für $y_2$\\
 				
 				$y_1 = c\cdot e^{\int b(x)dx}$\\
-				$y_2 = \int(a(x)e^{\int b(x)dx})dx\cdot e^{-\int b(x)dx}$\\
+				$y_2 = \int(a(x)e^{-\int b(x)dx})dx\cdot e^{-\int b(x)dx}$\\
```

Comment:
AFAIK this is not correct. This line should stay the same and the "-" should be added in y1, as done in my hotfix PR.